### PR TITLE
Add 'runbook' for adding admin users from freshdesk

### PR DESCRIPTION
**Pull request by yuvipanda**
Each time I handle a ticket on freshdesk, if one doesn't already exist, I'm going to try to write a clear, step by step 'runbook' that anyone doing support can follow whenever a specific kind of request comes in via freshdesk.

This is such a runbook for adding admins to a hub, for https://2i2c.freshdesk.com/a/tickets/1279 that led to https://github.com/2i2c-org/infrastructure/pull/3659

### CODE REVIEW

**Review by GeorgianaElena — APPROVED** [review]
Yay for having clear steps on as many support requests as possibile 🚀

Ideally, I think these runbooks should be linked from https://compass.2i2c.org/projects/managed-hubs/timeboxed-initial-ticket-evaluation and have that be the support triager's entrypoint into what can be solved immediately, without needing to take any decision and what should be escalated and tracked via GitHub issues.

File: docs/sre-guide/support/admin-users.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   >
+   > The usernames you provided have been made admins on the JupyterHub! You can verify
+   > this by going to the hub control panel at {{ hub-url }}/hub/home, and you should
+   > see an 'Admin' option in the top bar.
+   >
+   > Thanks!
+
+## Caveats & future work
+
+We don't have a clearly stated policy on wether admin access should
```
Comment (GeorgianaElena) [inline docs/sre-guide/support/admin-users.md:35]: Small typo:

```suggestion
We don't have a clearly stated policy on whether admin access should
```

**Comment by yuvipanda** [comment]
@GeorgianaElena oooh yes i agree that's a good place for them. I'd like for runbooks to be a top-level category, with a template. I'll work on that.

**Comment by yuvipanda** [comment]
Going to merge this and not wait on https://github.com/2i2c-org/team-compass/pull/797, because there's a big merge conflict there i need to fix.

**Pull request merged**